readthedocs/projects/querysets.py

```diff
@@ -17,7 +16,7 @@ class ProjectQuerySetBase(models.QuerySet):
     use_for_related_fields = True
 
     def _add_user_repos(self, queryset, user):
-        if user.has_perm('projects.view_project'):
```

Comment:
Does anyone have these permissions on .com or .org? I assume there was a reason we did it this way, but I don't remember :x

Reply:
Looks like we had a comment:

https://github.com/readthedocs/readthedocs.org/pull/2954/files#diff-05611882195df182851df952b312f031a4314bce04a104e10a09655e570515f3R82

Reply:
Only the build user has that permission set

```python
In [10]: User.objects.filter(user_permissions__isnull=False).distinct()
Out[10]: <QuerySet [<User: build>]>
```

That's in .org, in .com we have 0 users.